es6-lib/config/config.js

```diff
@@ -21,6 +21,9 @@ export default {
   //number of rows to buffer in object mode feature streams
   rowBufferSize: 2,
 
+  //maximum number of points to allow through the importer per row
+  maxVerticesPerRow: 1000000,
```

Comment:
It's annoying that JS and Java/Scala don't have the `1_000_000` syntax or something similar.